lib/pages/api/method/index.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
-  url: '/api/:apiMethod.html',
+  url: '/api/clearValue.html',
 
   sections: {
     container: {
       selector: '#apimethod-page',
       elements: {
-        header: '.page-header h2'
-      }
-    }
-  }
+        header: '.page-header h2',
+      },
```

Comment:
can you remove all the trailing commas? is this an editor setting?

Reply:
Removed, yes it happened due to prettier.